superset/async_events/api.py

```diff
@@ -33,6 +34,7 @@ class AsyncEventsRestApi(BaseSupersetApi):
     allow_browser_login = True
 
     @expose("/", methods=("GET",))
+    @statsd_metrics
     @event_logger.log_this
     @protect()
```

Comment:
Most API endpoint already use `@statsd_metrics`. In a case where this decorator is added to replace implicit `log_to_statsd=True`, the statsd metric 

- `superset_<Endpoint>` 

will be replaced by

- `superset_<Class>_<Endpoint>_success|warning|error`
- `superset_<Class>_<Endpoint>_time`
- `superset_<Class>_<Endpoint>_time_count`
- `superset_<Class>_<Endpoint>_time_sum`